onadata/apps/viewer/tests/test_export_builder.py

```diff
@@ -455,8 +455,10 @@ def test_xls_export_with_hxl_adds_extra_row(self):
         wb = load_workbook(temp_xls_file.name)
         children_sheet = wb.get_sheet_by_name("hxl_example")
         self.assertTrue(children_sheet)
+
         # we pick the second row because the first row has xform fieldnames
-        hxl_row = [a.value for a in children_sheet.rows[1]]
+        rows = [row for row in children_sheet.rows]
+        hxl_row = [a.value for a in rows[1]]
```

Comment:
Not longer really, this is in tests. `children_sheet.rows` due to package upgrade is now a generator/iterator and we can no longer use an index to get the second row.